tests/gitgitgadget.test.ts

```diff
@@ -240,7 +236,10 @@ to have included in git.git [https://github.com/git/git].`);
 
     const headCommit2 = await repo.revParse("HEAD");
     const patches2 =
-        await PatchSeries.getFromNotes(notes, pullRequestURL, description,
+        await PatchSeries.getFromNotes(notes, pullRequestURL,
+                                       "My first Pull Request!",
+                                       "Some Body <somebody@example.com>",
+                                       description,
```

Comment:
Only one of these two changed `getFromNotes()` calls should pass a valid submitter, the other one should pass `false` or `undefined` instead, to increase code coverage.

Reply:
I will add another patch with that.

Current patch doesn't touch the existing tests, only applying a fix elsewhere.